Tests/TuistCacheTests/Extensions/GraphTraverser+ExtrasTests.swift

```diff
@@ -0,0 +1,157 @@
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistSupport
+import XCTest
+@testable import TuistCache
+
+class GraphTraverserExtrasTests: XCTestCase {
+    func test_filterIncludedTargets_includeExcluded() throws {
```

Comment:
The name is not very descriptive of the scenario. I'd follow the convention:

```swift
func test_{function_name}_when_{scenario}
```

Reply:
Yeah, my intent here was to test all the possible scenarii involved using `filterIncludedTargets`. I can split them up